kvmagent/kvmagent/plugins/vm_local_volume_cache/command_wrapper/lvm.py

```
@@ -0,0 +1,546 @@
+    @staticmethod
+    def check_vg(vg_name):
+        # type: (str) -> bool
+        args = ["-qq", "--yes", vg_name]
+        cmd = shell.ShellCmd("%s %s" % (lvm.subcmd("vgck", timeout=5), ' '.join(args)))
+        cmd(is_exception=False)
+        if cmd.return_code != 0:
+            return False
+        return True
+
+    @staticmethod
+    def create_lv(lv_name, vg_name, type=None, stripes=None, stripesize=None, size=None, extents=None, tags=None):
```

Ruff (0.15.5)

[error] Function argument `type` is shadowing a Python builtin

(A002)